src/metadata/types/TableTypes.ts

```diff
@@ -1,4 +1,4 @@
 /**
  * Table type. Tables can be closure, junction,, etc.
  */
-export type TableType = "regular"|"view"|"junction"|"closure"|"closure-junction"|"entity-child";
+export type TableType = "regular"|"view"|"junction"|"closure"|"closure-junction"|"entity-child"|"embedded";
```

Comment:
I doubt we need it. Embeds already supported and everything work correctly without this new table type. You just need properly convert entity schema definitions into metadata just like what decorator-based stuff does.

Reply:
@pleerock I did a lot of changes. Could you check again?